packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -532,9 +532,9 @@ export class SchemaPublisher {
         selector,
       });
 
-    const schemaVersionContracts = comparedSchemaVersion
+    const schemaVersionContracts = latestSchemaVersion
```

Comment:
I don't think we should use latestSchemaVersion here. We decided that schema versions will always be compared against the last composable version

Reply:
The latest schema version contracts should only be used for the checksum.